layers/+misc/debug/config.el

```diff
@@ -0,0 +1,8 @@
+(defvar debug-mode 'gdb
+  "Options are `gdb' to run specific debugger.")
+
+(defvar debug-prog nil
+  "Variable that holds the path to the debugger executable.")
+
+(defvar debug-target nil
+  "Executable that should be debugged.")
```

Comment:
If the idea is that these variables should be buffer-local¸ probably the C-C++ layer needs to set them buffer-locally. If that is not the idea I'd like to know why (seems reasonable that in the future other modes will use other debug programs.)